src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.test.tsx

```diff
@@ -42,10 +42,15 @@ const getComponentProps = (
 
 jest.mock('../../lib/okta-i18n', () => ({
   useTranslation: () => ({
-    t: jest.fn()
-      .mockReturnValueOnce('Phone Number')
-      .mockReturnValueOnce('Country')
-      .mockReturnValueOnce('Extension'),
+    t: (val: string) => {
+      if (val === 'country.label') {
+        return 'Country';
+      }
+      if (val === 'phone.extention.label') {
+        return 'Extension';
+      }
+      return val;
+    },
```

Comment:
Do we need to mock the translations out still?

Reply:
Nope, this was before I merged the latest from 0.0.